mmv1/third_party/terraform/tests/resource_compute_region_security_policy_rule_test.go.erb

```diff
@@ -0,0 +1,326 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"testing"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccComputeRegionSecurityPolicyRule_regionSecurityPolicyRuleBasicUpdateExample(t *testing.T) {
```

Comment:
```suggestion
func TestAccComputeRegionSecurityPolicyRule_regionSecurityPolicyRuleBasicUpdate(t *testing.T) {
```

Reply:
We should omit the `Example` suffix from handwritten tests- they're not testing docs examples.

Reply:
Done